assets/less/components/links.less

```diff
@@ -73,6 +73,29 @@ a {
   }
 }
 
+.social-link-large {
+  &:extend(.social-link, .font-regular, .bg-grey, .text-blue, .font-semibold, .shadow-md, .mb-4);
```

Comment:
let's use a `text-blue-dark` for these, as well as the background color on rollover in order to guarantee AA readability.
The firefox inspector will also complaint if the AA is below 4points.